conf/application.conf

```diff
@@ -1,49 +1,124 @@
 # This is the main configuration file for the application.
 # ~~~~~
 
-# Secret key
+
+# PRODUCTS
 # ~~~~~
-# The secret key is used to secure cryptographics functions.
-# If you deploy your application to several instances be sure to use the same key!
-play.crypto.secret="changeme"
-play.crypto.secret=${?APPLICATION_SECRET}
 
-# Injection configuration
+# Product attribute list that is displayed for a specific product variant (this does not allow to select different
```

Comment:
So I changed it to `name`, which is the official name we use everywhere. Maybe to avoid confusion with `label` I may add some clarification if I see somebody has trouble with it, but now I'm going to see how it goes like this.
I don't want to use `attribute field name` because to me it sounds too much like `custom field name` and I think it has the same chances to confuse than to help.